src/main/java/nextstep/subway/line/domain/model/Sections.java

```diff
@@ -0,0 +1,110 @@
+package nextstep.subway.line.domain.model;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+import javax.persistence.CascadeType;
+import javax.persistence.Embeddable;
+import javax.persistence.FetchType;
+import javax.persistence.OneToMany;
+
+import nextstep.subway.common.exception.ErrorMessage;
+import nextstep.subway.common.exception.InvalidArgumentException;
+import nextstep.subway.common.exception.OptionalException;
+import nextstep.subway.station.domain.model.Station;
+
+@Embeddable
+public class Sections {
+    @OneToMany(
+        fetch = FetchType.LAZY,
+        mappedBy = "line",
+        cascade = { CascadeType.PERSIST, CascadeType.MERGE },
+        orphanRemoval = true
+    )
+    private List<Section> values;
+
+    public Sections() {
+        values = new ArrayList<>();
+    }
+
+    public Section add(Line line, Station upStation, Station downStation, Distance distance) {
+        verifySectionDockingPoint(upStation).verify();
+        verifyAlreadyRegisteredStationInSection(downStation).verify();
```

Comment:
```suggestion
        if(addable(upStation, downStation){
            throw new xxxxException();
        }
```
그냥 이정도로 마무리 지을수도 있지 않을까 싶어요 ㅎㅎ